cmd/oras/internal/option/remote.go

```diff
@@ -244,7 +253,19 @@ func (opts *Remote) parseCustomHeaders() error {
 
 // Credential returns a credential based on the remote options.
 func (opts *Remote) Credential() auth.Credential {
-	return credential.Credential(opts.Username, opts.Password)
+	if opts.IdentityToken != "" {
+		// If IdentityToken is provided, use it as the credential without a username
+		return auth.Credential{
+			Username: "",
+			Password: opts.IdentityToken,
+		}
+	} else {
+		// If IdentityToken is not provided, use the username and password as credentials
+		return auth.Credential{
+			Username: opts.Username,
+			Password: opts.Password,
+		}
+	}
```

Comment:
This block of code should be reverted since `credential.Credential()` already handles everything.

Reply:
Okay, will fix it.